modules/serverless-resolving-operator-upgrade-failure.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-resolving-operator-upgrade-failure_{context}"]
+= Resolving {ServerlessOperatorName} upgrade failure
+
+In some cases, such as when you do manual uninstalls and reinstalls, you might encounter an error when upgrading {ServerlessOperatorName}. In this situation, you need to manually reinstall {ServerlessOperatorName}.
+
+.Procedure
+
+. Identify the version of {ServerlessOperatorName} that was installed originally using the `project.yaml` files for the different releases.
```

Comment:
I'm not very happy to point customers to the "development" project.yaml from our repository. We could probably achieve the same by pointing them to release notes.

If the installed KnativeServing version is v1.5.0 where the version has this pattern: vMAJOR.MINOR.PATCH. Then searching through release notes for MAJOR.MINOR version of Knative Serving will give them the version of Serverless. 
For example, [release notes for 1.26](https://58238--docspreview.netlify.app/openshift-enterprise/latest/serverless/serverless-release-notes.html#serverless-rn-1-26_serverless-release-notes) have this: `OpenShift Serverless now uses Knative Serving 1.5.`